docs/iris/src/whatsnew/contributions_1.11/bugfix_2016-Oct-06_is_circular.txt

```diff
@@ -0,0 +1,2 @@
+- Fixed a bug where the :func:`iris.util._is_circular` would return false where 
```

Comment:
Fine, I'd prefer if we tried to stick to the same throughout, but depending on restructured-text backend it is worst case a warning if you mix them, so I guess it doesn't matter as much as I thought.